.github/workflows/release-candidate.yaml

```diff
@@ -110,13 +107,9 @@ jobs:
           echo "latest_commit_sha=$(git log -n 1 --pretty=format:"%H")" >> $GITHUB_ENV
 
       - name: Create GitHub Release
-        uses: actions/create-release@latest
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
```

Comment:
Replacing this because it required us to do a duplicate checkout of the registry at the root.